### DISCUSSION TITLE
Use Anthropic models through Google/Vertex api and keys

### DISCUSSION THREAD

**Discussion by Spyros-Vl** [danny-avila/LibreChat#3298] [Help Wanted]
Hello everyone, I am just curious if any of you tried to use the Anthropic models through Vertex AI and if this is possible with the librechat setup. If yes can we just include the Anthropic model in the GOOGLE_MODELS .env variable and use them?

**Comment by zazer0** [comment]
Did you figure this out?

**Reply by Spyros-Vl** [reply]
I just tried to add the anthropic model into the google models in the env file after I apply for it in Vertex but this didnt work. Other than that I havent tried anything else.

**Reply by ChristianPilot** [reply]
> I just tried to add the anthropic model into the google models in the env file after I apply for it in Vertex but this didnt work. Other than that I havent tried anything else.

Did you ever solve this issue?

**Comment by r1di** [comment]
did you guys found a solution to this?

**Reply by otinas** [reply]
Is there any upside other than the free-trial GCP provides? Not saying we don't need the functionality, but just curious.

**Comment by Ziyann** [comment]
I submitted a PR draft for this, comments are welcome: https://github.com/danny-avila/LibreChat/pull/9798.
